### ISSUE TITLE
Provide GPS time over the local network as an NTP server

### ISSUE THREAD

**Issue by elafargue** [jks-prv/Beagle_SDR_GPS#150]
Since KiwiSDR can use GPS as its clock source, it would be a great NTP time source for a local network. This would require exposing the GPS time to a local NTP server. Most ntpd implementations seem to use a 1PPS output from the GPS, so I don't know whether this could apply to the KiwiSDR, but why not look into it...

Examples of prior art: http://www.catb.org/gpsd/gpsd-time-service-howto.html or http://www.satsignal.eu/ntp/Raspberry-Pi-NTP.html

**Comment by jks-prv** [comment]
Yes, certainly should be possible. You'd want to make sure there is really good outlier filtering to keep the odd bad GPS solution from getting into NTPd. The Kiwi code has some of this now, but it is pretty simple. A good project for someone! lol
